source/edge-server/configure.txt

```diff
@@ -551,34 +593,111 @@ look similar to:
 
 For a more complex query, where you want to pass different queries for 
-multiple collections, you might want to pass the query as JSON. This enables
-you to pass a different query for each schema title as a key-value pair. For
-example:
+multiple collections, you have two options:
 
-.. code-block:: shell
+- Pass multiple ``--query`` flags, one for each collection
+- Pass a path to a file where you define the queries as JSON
 
-   edgectl --query={ \
-         "Project": "facility_id == 42", \
-         "Item": "priority >= 3" \
-      }
-
-In this example, we have three collections: ``Project``, ``Item``, and 
-``Team``, whose collection names match their schema titles. This query syncs 
+In these examples, we have three collections: ``Project``, ``Item``, and 
+``Team``, whose collection names match their schema titles. These queries sync 
 data with two of the collections: ``Item`` and ``Project``. The 
 ``Project`` collection has a list property that represents a 
 :ref:`to-many relationship <relationships-to-many>` to the ``Item`` 
 collection. We don't want to sync the ``Team`` collection to the Edge Server 
-instance. By omitting this schema title from the query, we don't sync any of 
+instance. By omitting this schema title from the queries, we don't sync any of 
 its data to the instance.
 
 With these queries, we are only syncing ``Project`` documents for facility 42. 
 Additionally, we are only syncing ``Item`` documents where the ``priority`` 
 field's value is greater than or equal to ``3``. In this way, you can combine 
 queries to sync only the documents that are relevant to a given Edge Server 
 instance.
+
+Pass Multiple Query Flags
++++++++++++++++++++++++++
+
+You can pass multiple query flags with ``edgectl config`` to provide queries
+for multiple collections.
+
+.. code-block:: console
+
+   edgectl config --query="Project: facility_id == 42" --query "Item: priority >= 3"
+
+Pass Queries as a JSON File
++++++++++++++++++++++++++++
+
+You can pass a path to a JSON file that defines many different queries. For
+example:
+
+.. code-block:: console
+
+   edgectl config --query-file="/Users/my.user/edge-server/query.json"
+
+Where the JSON file contains:
+
+.. code-block:: json
+
+   {
+     "Project": "facility_id == 42",
+     "Item": "priority >= 3"
+   }
+
+.. _multiple-edge-servers-on-host:
+
+Run Multiple Edge Server Instances on a Host
+--------------------------------------------
+
+While your Edge Server architecture is in development, you may want to run
+multiple Edge Server instances on a single host - for example, your local
+machine. You can do this using the ``edgectl`` profile functionality.
+
+When you create an Edge Server instance, ``edgectl`` automatically creates
+a default profile for you. You never have to specify the profile while there
+is only one profile.
+
+You can view the profiles, and their associated configuration files, in:
+
+.. code-block:: console
```

Comment:
i'm going to do my best to get some better profile management in by monday too. this will allow
```
edgectl profile list
edgectl profile delete <profile>
edgectl profile inspect <profile>
edgectl profile set-default <profile>
```

when you have a default profile, you dont need to pass `--profile` to all commands

Reply:
Given time constraints for prepping the feature branch docs for publication, I'll defer the new `profile` functionality to a fast-follow that we can publish after the publishing freeze is over.